llvm/lib/Target/AMDGPU/VOP3Instructions.td

```diff
@@ -570,6 +570,36 @@ def VOP3_CVT_PK_F8_F32_Profile : VOP3_Profile<VOP_I32_F32_F32, VOP3_OPSEL> {
   let HasExtVOP3DPP = 1;
 }
 
+def VOP3_CVT_PK_F8_F32_Profile_fake16 : VOP3_Profile_Fake16<VOP_I16_F32_F32, VOP3_OPSEL> {
```

Comment:
I'd prefer avoid this and let these to be generated automatically, then use VOP3_Profile_True16/VOP3_Profile_Fake16 wrappers directly or indirectly. It should probably already work, if not some changes may need to be pulled from downstream.

Reply:
Hi Stanislav, I do not fully understand this. Are you recommending not to inherit from  VOP3_Profile_Fake16?